mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/embeddable/ElementCollectionOrderWithSessionTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.mongodb.test.embeddable;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.Transaction;
+import org.hibernate.ogm.OgmSession;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author Davide D'Alto
+ */
+@TestForIssue(jiraKey = "OGM-1238")
+public class ElementCollectionOrderWithSessionTest extends OgmTestCase {
```

Comment:
I think so, we are testing two different interfaces and the test classes are simple. I don't see where's the problem. Potentially, comsething could happen between EntityManager and Session that causes a test to fail.